packages/pg/lib/client.js

```diff
@@ -718,11 +796,16 @@ class Client extends EventEmitter {
   }
 
   connect(callback) {
-    if (!this.connectionParameters.loadBalance) {
+    if (this.connectionParameters.loadBalance === 'false') {
       logger.silly("Loadbalance is false, falling to upstream behaviour")
       return this.nowConnect(callback)
     }
     lock.acquire().then(() => {
+      logger.silly("loadBalance: " + this.connectionParameters.loadBalance)
+      logger.silly("topologyKeys: " + this.connectionParameters.topologyKeys)
+      logger.silly("ybServersRefreshInterval: " + this.connectionParameters.ybServersRefreshInterval)
+      logger.silly("fallbackToTopologyKeysOnly: " + this.connectionParameters.fallbackToTopologyKeysOnly)
+      logger.silly("failedHostReconnectDelaySecs: " + this.connectionParameters.failedHostReconnectDelaySecs)
```

Comment:
For line# 823 (new code):  else block should also throw err? At the least log a msg if not needed with an explanation

Reply:
Added a ToDo comment and also a log.